src/style.css

```diff
@@ -1,3 +1,63 @@
+:root {
+  --accent-color: #00acdc;
+}
+
+html,
 body {
   margin: 0;
+  padding: 0;
+}
+
+header {
```

Comment:
not fixed
![image](https://github.com/mate-academy/layout_moyo-header/assets/70009205/e7af98b1-f67d-4915-9fd4-cd9e75ce4453)